### DISCUSSION TITLE
Is there a way to forbid/limit testcontainers from binding to a range of ports?

### DISCUSSION THREAD

**Discussion by anthonyma94** [testcontainers/testcontainers-node#759] [Q&A]
For example, when using `withExposedPorts()`, I want testcontainers to either *not* bind to ports 3000-3999, or to *only* bind to ports 10000-20000. Is this possible?

**Comment by cristianrgreco** [comment]
Hi @anthonyma94, there isn't. It seems like a niche use case.
